ansible/www-standalone/tasks/site-setup.yaml

```diff
@@ -110,11 +110,9 @@
     dest: /etc/crontab
     line: "{{ item }}"
   with_items:
-    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
-    - '*/5 *   * * *   dist    /home/staging/tools/promote/promote_nightly.sh iojs'
-    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
-    - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh iojs'
-    - '*  *    * * *   root    /home/nodejs/cdn-purge.sh'
+    - '*/30 *    * * *   dist    /home/staging/tools/promote/promote_nightly.sh nodejs'
+    - '*/5  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
+    - '*/5  *    * * *   root    /home/nodejs/cdn-purge.sh'
```

Comment:
The actual cache purge script, it looks at its "queue" (just a temp file that any one of the above processes might create) and if there's a purge queued then it does a `curl` to CloudFlare to request a full purge. By reducing from every 1 minute to every 5 minutes we're reducing the potential for no caching at the edges.